### PR TITLE
Issue #1641 css

### PR DESCRIPTION
**Pull request by stefanhaerter**
Closing #1641

### CODE REVIEW

File: var/httpd/htdocs/js/Core.UI.InputFields.js [hunk 1]
```diff
@@ -2622,7 +2622,11 @@ Core.UI.InputFields = (function (TargetNS) {
                         }
                     }
                     CheckAvailability($SelectObj, $SearchObj, $InputContainerObj);
+                    var SelectObjVisibility = $SelectObj.css('display');
+                    console.log(SelectObjVisibility);
```
Comment (svenoe) [inline var/httpd/htdocs/js/Core.UI.InputFields.js]: ...

**Comment by svenoe** [comment]
We will let this sit for a while, to look for the reason it specifically doesn't work in the AdminDynamicField screens, but does at other places, to see whether we find a more local fix.

**Comment by stefanhaerter** [comment]
Checking the following places:
- [ ] AgentTicket(Zoom|Email|Process)
- [ ] DynamicField Configurations
- [ ] Customer Screens (Searching where dropdowns are used there)

**Comment by stefanhaerter** [comment]
Closing this in favor of #2530

**Pull request closed without merging**